applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_solver.py

```diff
@@ -30,6 +30,7 @@ def GetDefaultSettings(cls):
             "time_order": 2,
             "time_scheme": "bdf2",
             "compute_reactions": false,
+            "analysis_type": "non_linear",
```

Comment:
this should really be refactored to become part of the baseclass! (separate PR, just mentioning it again)

Reply:
This solver is legacy code. I will upgrade It as soon as I finish the explicit one.

Reply:
I meant this setting in general, since it is used and needed in the baseclass thats where it should be (i.e. the `FluidSolver` should have settings itself, but I guess this is already on the todo list

Reply:
Agree... The base solver has no settings yet ... This is in the TODOs of the fluid project.